openapi/openapi.bzl

```diff
@@ -4,8 +4,8 @@
 
 load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
 
-
-def openapi_repositories(swagger_codegen_cli_version = "2.2.2", swagger_codegen_cli_sha256 = "d4e640503482a423ae97e3f487347c9e08cc0e34d90e360ed7de987cfad24299", prefix = "io_bazel_rules_openapi"):
+# https://swagger.io/docs/open-source-tools/swagger-codegen/
+def openapi_repositories(swagger_codegen_cli_version = "2.4.10", swagger_codegen_cli_sha256 = "cfafe385f2117044adb5b6f29743a832052e12431a93a462f8965e9b8d786b4e", prefix = "io_bazel_rules_openapi"):
```

Comment:
Don't forget first update, test, and publish the custom code generation module meetup manages for generating clients over [here](https://github.com/meetup/meetup-scala-generator) before folding these changes into the classic repo

Reply:
Sounds good!!

Reply:
Based on the scala-generator test, I will just upgrade to 2.4.9 for now